src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx

```diff
@@ -366,9 +341,54 @@ The following table shows the default retention settings for our [New Relic One
   </tbody>
 </table>
 
+## Adjust retention values [#adjust-retention]
+
+You can adjust your retention periods to better support your data needs and meet compliance requirements. You might reduce retention values to optimize query performance. Or, you might increase retention for some data types to support more granular analysis over longer periods of time.  
+
+The ability to edit data retention periods requires a specific [edition of New Relic](https://newrelic.com/pricing).
+
+* Legacy accounts must have Insights Pro. 
+
+* New Relic One accounts have these edition considerations:
+
+  * Standard edition: your data retention is fixed, and falls between 8 and 395 days (about 13 months) of retention, depending on which capabilities of the New Relic platform you’re using. The prescribed retention levels ensure that you can explore your data and experience the value of New Relic. If you need to raise some of the retention limits for your particular business needs, you can purchase additional retention.
+  * Pro or Enterprise: If after you finalize your contracted retention you realize you need to make changes, you can adjust your telemetry data retention settings yourself.
+
+### User roles and retention
+
+In order to edit retention, a V1 user must be the **Owner** or have the **Data Retention Manager** add-on role. A V2 user must have the **billing_user** role. 
+
+### Reduce retention values
+
+When you reduce the number of days that specific data sources are retained, existing data of that source type won't immediately disappear and might be visible and returned in queries for up to several weeks. We have safeguards in place to ensure you don't accidentally delete data, but this also means that reducing retention values isn't a good strategy for quickly [removing data](#data-deletion).
+
+That said, your change is registered in the system within 12 hours and new data of the adjusted source type will be retained for the new duration. 
+
+<Callout variant="tip">
+Reducing data retention below your contracted retention values will not reduce New Relic’s data ingest charges. Conversely, increasing retention for targeted telemetry data types is likely to trigger a conversation with New Relic about adjusting your data ingest charges.
+  </Callout>
+
+## Copy retention values
+
+If you have permission to edit retention settings, you can also copy retention values from one child account to other child accounts under the same parent account. To do this, from within a child account, select the **Copy settings to other accounts** link, and then select the child accounts you want to apply the retention values to.
```

Comment:
Is parent/child terminology still used? I thought they were switching to account/sub-account but I may be wrong.

Reply:
it's opposite: it was master/sub-account and now it's parent and child account. But in v2 world, these things are much less important and not exposed to v2 customers (they have no way of understanding what's a parent account or child account, it's primarily under the hood). Might be worth us talking about that, Michelle, as reading this, I'm not sure as a v2 user what this would mean. But feel free to roll it out, maybe just something for me to review later, and I'll make note of it.